packages/node-app/src/app.ts

```diff
@@ -8,8 +10,35 @@ import { router as LocationRoutes } from './routes/location.routes';
 
 export const app = express();
 
+/*
+ * The option to try out the API via Swagger won't be available
+ * until the authentication is independent from the frontend
+ * TODO: https://runtime-revolution.atlassian.net/browse/WEB3-61
+ */
```

Comment:
```suggestion
/*
 * The option to try out the API via Swagger won't be available
 * until the authentication is independent from google auth
 * TODO: https://runtime-revolution.atlassian.net/browse/WEB3-61
 */
```

Also two points regarding the ticket description:
- it is possible to use google auth in postman
- do we want to support user auth without google auth on the app? What's the idea exactly, could you complete the ticket description please?

Reply:
I'm not 100% sure because I haven't explore the code yet, but my idea is to keep the google authentication and just add the necessary endpoints in the API to get the tokens without the need to use the frontend

Reply:
Could we update the comment to include the suggested change? I think it is more precise since the dependency is not from frontend specifically, it's from google authentication.

Reply:
But the API is still going to be dependent on the google authentication. The only difference is that it will have the necessary endpoint to do all the authentication process without the need to use the frontend